draft-ietf-quic-recovery.md

```diff
@@ -467,14 +467,14 @@ Handshake packet number spaces, the max_ack_delay is 0, as specified in
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-A sender computes its PTO timer every time an ack-eliciting packet is sent.
+A sender resets its PTO timer every time an ack-eliciting packet is sent.
```

Comment:
I think computes is more accurate in this case, because the timer may not change if another packet number space will fire earlier.

Reply:
On a quick read it's actually not clear from the beginning that there is only one timer and not a separate timer for each packet. So maybe "recomputes" catches that? Or we could add a separate (half-)sentence.

Reply:
PTO timer is singular, so....?

Reply:
I pulled this into #3552 with some tweaks, PTAL.